Unix/tests/http/test_httpclient.cpp

```diff
@@ -141,6 +146,33 @@ void _StrandTestClientPost( _In_ Strand* self_, _In_ Message* msg )
     else
         s_charset = "";
 
+    if (request->headers->authorization)
```

Comment:
aren't you going to want to crack open the basic auth header to validate it is encoded properly in there? That would be more complete.